TypeScript createInstance() generates invalid code for enums

## Description

The \createInstance()\ method in TypeScript generated code uses \.values\ on enum types, but TypeScript enums don't have a \.values\ property.

## Error

\\\
src/TestProjectData/Net/Fabrikam/Telemetry/Channel.ts(112,25): error TS2339: Property 'values' does not exist on type 'typeof ChannelType'.
\\\

## Reproduction

Generate TypeScript code for any schema with enums (e.g., fabrikam-motorsports.xreg.json from xregistry/codegen).

## Expected

The generated code should use \Object.values(EnumType)\ instead of \EnumType.values\.

## Version

avrotize 2.20.2

--- a/avrotize/avrotots.py
+++ b/avrotize/avrotots.py
@@ -266,9 +266,9 @@ class AvroToTypeScript:
             first_type = field_type.split('|')[0].strip()
             return self.generate_test_value(first_type, is_enum)
         
-        # Handle enums - use first value (will be set via template)
+        # Handle enums - use first value with Object.values()
         if is_enum:
-            return f'{field_type}.values()[0]'
+            return f'Object.values({field_type})[0] as {field_type}'
         
         # Handle primitive types
         primitive_values = {

